### DISCUSSION TITLE
[Quicker 动作]一键保存到稍后读

### DISCUSSION THREAD

**Discussion by Joel2561** [Kenshin/simpread#2217] [服务]
# 描述

> 简悦拥有众多保存到稍后读的途径，例如通过右键选单将当前页面加入稍后读，或者通过快捷键`DD`直接将当前页面保存到稍后读，但简悦新出了api，于是追求效率的我（其实是闲的）写了一个quicker动作，顺手分享出来。

> 通过api可以实现在任何场景下（而不仅仅是在浏览器里），只要你有一个链接需要稍后阅读，就可以通过quicker把它添加到简悦稍后读中。

# 使用

1. 获取Quicker动作指令并增加至 Quicker 中, 获取地址 👉 [保存到稍后读](https://getquicker.net/sharedaction?code=6bdc2ec2-026d-44d2-c3a3-08d908802222)
2. 找到购买简悦高级账户时的licence（激活码）和自己的UID（选项页—账户），进入[简悦开放平台]，(https://simpread.pro/developer/) 复制token，打开quicker动作页编辑。
3. 在右侧变量名token里找到默认值，填入token。
![image](https://user-images.githubusercontent.com/77143461/119140316-9560fa00-ba76-11eb-823d-f1dce9e4f443.png)

  4.选中任意含有需要做稍后读的链接，运行quicker动作，完成。

# 个性化

如果不想输入描述，标签或备注，请停用编辑页中三个<用户输入>步骤，并双击编辑页HTTP请求，自行删除下面的四行（倒数第四行的逗号也要删掉）

                                      ,

    "desc": "{description}",
    "tags": "{tag}",
    "note": "{note}"

# 动作原理
利用 Quicker 获取选中文本/获取剪切板中的文本 → 将文本中的链接提取出来处理为纯文本 → 获取链接对应的标题 → 请求api → 返回状态码，动作结束。

# 注意事项
此动作只能添加一个链接，后面如果大家有需求会做批量收藏。

**Comment by Kenshin** [comment]
哈哈，大爱啊！ 👍🏻

**Reply by GaussianZhao** [reply]
谢谢大佬

**Comment by Sunsh4j** [comment]
> 哈哈，大爱啊！ 👍🏻

大佬，我想仿照这个示例用简悦的api导入到Notion里，但是现在api仅支持读写，能否对导入Notion也支持一下🧡🧡
